pyproject.toml

```diff
@@ -92,28 +129,15 @@ platforms = ["linux-64", "win-64", "osx-arm64"]
 Ragna = { path = ".", editable = true }
 
 [tool.pixi.environments]
-default = { features = ["py310"], solve-group = "default" }
-default-py310 = { features = ["py310"], solve-group = "default" }
-default-py311 = { features = ["py311"], solve-group = "default-py311" }
-default-py312 = { features = ["py312"], solve-group = "default-py312" }
-default-py313 = { features = ["py313"], solve-group = "default-py313" }
-default-all = { features = ["all", "py310"], solve-group = "default" }
-default-all-py310 = { features = ["all", "py310"], solve-group = "default" }
-default-all-py311 = { features = ["all", "py311"], solve-group = "default-py311" }
-default-all-py312 = { features = ["all", "py312"], solve-group = "default-py312" }
-default-all-py313 = { features = ["all", "py313"], solve-group = "default-py313" }
-dev = { features = ["dev", "py310"], solve-group = "default" }
-dev-py310 = { features = ["dev", "py310"], solve-group = "default" }
-dev-py311 = { features = ["dev", "py311"], solve-group = "default-py311" }
-dev-py312 = { features = ["dev", "py312"], solve-group = "default-py312" }
-dev-py313 = { features = ["dev", "py313"], solve-group = "default-py313" }
-dev-all = { features = ["all", "dev", "py310"], solve-group = "default" }
-dev-all-py310 = { features = ["all", "dev", "py310"], solve-group = "default" }
-dev-all-py311 = { features = ["all", "dev", "py311"], solve-group = "default-py311" }
-dev-all-py312 = { features = ["all", "dev", "py312"], solve-group = "default-py312" }
-dev-all-py313 = { features = ["all", "dev", "py313"], solve-group = "default-py313" }
-
-[tool.pixi.tasks]
+base-py310 = { features = ["dev", "py310"], solve-group = "py310" }
+base-py311 = { features = ["dev", "py311"], solve-group = "py311" }
+base-py312 = { features = ["dev", "py312"], solve-group = "py312" }
+base-py313 = { features = ["dev", "py313"], solve-group = "py313" }
+all-py310 = { features = ["dev", "py310", "all"], solve-group = "py310" }
+all-py311 = { features = ["dev", "py311", "all"], solve-group = "py311" }
+all-py312 = { features = ["dev", "py312", "all"], solve-group = "py312" }
+all-py313 = { features = ["dev", "py313", "all"], solve-group = "py313" }
+all-docker = {features = ["py313", "all"], solve-group="py313"}
```

Comment:
Reduce cuts down the number of envs from 20 to 9:

- Except for the env that we use as part of our Dockerfile (now named `all-docker`), we don't need the `default*` envs as we need dev dependencies for everything else.
- The `default`, `default-all`, `dev`, and `dev-all` envs are not required as they are just "aliases", while taking up real space.